workos/directory_sync.py

```diff
@@ -439,19 +439,19 @@ def list_directories(
 
     def list_directories_v2(
         self,
-        domain=None,
-        search=None,
-        limit=None,
-        before=None,
-        after=None,
-        organization=None,
-        order=None,
+        domain: str = None,
+        search: str = None,
+        limit: int = None,
+        before: str = None,
+        after: str = None,
+        organization: str = None,
+        order: Order = None,
```

Comment:
I'm new to python type hints, but is this correct? Coming from a TS world something like `foo(domain: string = null)` is not valid.

Reply:
I'm still learning whether that's OK in Python. It's OK to set a type hint without initializing a variable.


... ah, OK, looks like it's OK to initialize with `None`

Reply:
We could definitely make this better with `Optional`, which I'll do.

Reply:
`Optional` does sound more correct here. I think. Says the person that doesn't actually write anything in Python :P